Rakefile

```diff
@@ -12,6 +12,102 @@
+# | 3  | regen_all_profiles              | Regenerates all the profiles for all the services. This task has regen_rollup_profile and        |
+# |    |                                 | regen_individual_profiles as prereqs.                                                            |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 4  | clean_generated                 | Cleans all the generated folders in multiple SDK versions.                                       |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 5  | regen_rollup_profile            | Regenerates all profiles within the rollup gem. At the end, this task calls                      |
+# |    |                                 | regen_rollup_require_files as an enhancement. This task has clean_rollup_profiles as prereq.     |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 6  | regen_individual_profiles       | Regenerates all profiles with all the services. At the end, this task calls                      |
+# |    |                                 | regen_individual_require_files as an enhancement. This task calls clean_individual_profiles      |
+# |    |                                 | as prereq.                                                                                       |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 7  | clean_all_profiles              | Cleans all profiles inside the rollup gem and individual gems.This task has clean_rollup_profiles|
+# |    |                                 | and clean_individual_profiles as prereqs.                                                        |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 8  | clean_rollup_profiles           | Cleans all profiles inside the rollup gem.                                                       |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 9  | clean_individual_profiles       | Cleans all profiles inside individual gems.                                                      |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 10 | regen_all_require_files         | Regenerates the require files for the rollup gem and individual gems. This task has              |
```

Comment:
I don't think it's clear what "require_files" are, can we add a definition of that somewhere? or a pointer to an example? you could add a reference here like (*) or (1) and explain below

Reply:
Done for the first one. WIP on CI modification